apps/test/unit/lib/kits/maker/ThermometerTest.js

```diff
@@ -1,8 +1,12 @@
 /** @file Test the Thermometer controller wrapping playground-io Thermometer */
 import {expect} from '../../../../util/deprecatedChai';
```

Comment:
If it's not a time sink, can you replace `deprecatedChai` with `reconfiguredChai` while you're here?